packages/cli/src/hooks/init/init-config.ts

```diff
@@ -1,4 +1,4 @@
-import { Hook } from '@oclif/config'
+import type { Hook } from '@oclif/core'
```

Comment:
Was this intentional?

Reply:
Yes, I thought I'd change this to a [type-only import](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-8.html#type-only-imports-and-export).